internal/app/go-filecoin/internal/submodule/storage_mining_submodule.go

```diff
@@ -35,7 +37,20 @@ type StorageMiningSubmodule struct {
 // NewStorageMiningSubmodule creates a new storage mining submodule.
 func NewStorageMiningSubmodule(minerAddr address.Address, ds datastore.Batching, s sectorbuilder.Interface, c *ChainSubmodule, m *MessagingSubmodule, mw *msg.Waiter, w *WalletSubmodule, stateViewer *appstate.Viewer) (*StorageMiningSubmodule, error) {
 	minerNode := storageminerconnector.NewStorageMinerNodeConnector(minerAddr, c.ChainReader, c.State, m.Outbox, mw, w.Wallet, stateViewer)
-	storageMiner, err := storage.NewMiner(minerNode, ds, s, minerAddr)
+
+	// The amount of epochs we expect the storage miner to take to replicate and
+	// prove a sector. This value should be shared with the storage miner side
+	// of go-fil-markets.
+	//
+	// TODO: What is the correct value for proving delay given 32GiB sectors?
+	provingDelay := abi.ChainEpoch(2 * 60 * 24)
```

Comment:
See the miner.MaxSealDuration table in the actor code. It's used in `verifySeal` in the miner actor. 

I think that table is only for the delay between pre- and post-commit, so this might be a larger number. To leave room for versioning of that number it's probably best that it's recapitulated here though, but maybe leave a pointer to the actor policy.

Reply:
@anorth - Very good. Thanks for the pointer to `miner.MaxSealDuration`. I'll leave a pointer to that value in this code.